operation/fhir-operation-reindex/src/main/java/com/ibm/fhir/operation/reindex/ReindexOperation.java

```diff
@@ -71,7 +72,7 @@ protected Parameters doInvoke(FHIROperationContext operationContext, Class<? ext
         // Allow only POST because we're changing the state of the database
         String method = (String) operationContext.getProperty(FHIROperationContext.PROPNAME_METHOD_TYPE);
         if (!"POST".equalsIgnoreCase(method)) {
-            throw new FHIROperationException("HTTP method not supported: " + method);
+            throw FHIROperationUtil.buildExceptionWithIssue("HTTP method not supported: " + method, IssueType.NOT_SUPPORTED);
```

Comment:
It reminds me that I'd like to move this check to our operations framework and base it on the value of this element:  https://www.hl7.org/fhir/operationdefinition-definitions.html#OperationDefinition.affectsState

Reply:
https://github.com/IBM/FHIR/issues/2504